packages/zod-openapi/package.json

```diff
@@ -52,7 +51,7 @@
   },
   "dependencies": {
     "@asteasolutions/zod-to-openapi": "^7.1.0",
-    "@hono/zod-validator": "npm:0.4.2"
+    "@hono/zod-validator": "workspace:^"
```

Comment:
Now that CI builds dependencies of a workspace, the `workspace:` protocol can be used to reference `@hono/zod-validator` directly instead of from npm